.github/workflows/vcradler-workflow.yaml

```diff
@@ -0,0 +1,31 @@
+name: build and run Dockerfile for vcralder
+on:
+    pull_request:
+        branches:
+            - master
+
+        paths:
+            - '/individual-work/vcradler/**'
+            - '.github/workflows/vcradler-workflow.yaml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+
+    steps:
+      - name: Fetch my code repositoy
+        uses: actions/checkout@v2
+
+      - name: Please fail, code!
```

Comment:
This is well and good for our current purposes (and a bit comical to include, so I don't mind), but please ensure this is removed in a future iteration